layers/+spacemacs/spacemacs-evil/packages.el

```diff
@@ -152,6 +154,9 @@
         (evil-goggles-mode -1))
       (spacemacs|hide-lighter evil-goggles-mode))))
 
+(defun spacemacs-evil/init-evil-exwm-state ()
+  (use-package evil-exwm-state))
```

Comment:
Can we also make depending on these packages conditional on having enabled the layer?

Reply:
Will do.